flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/CliFrontend.java

```diff
@@ -114,6 +122,22 @@ static CliExecutor createExecutor(CommandLine commandLine) throws Exception {
                 savepointSettings);
     }
 
+    private static void overrideFlinkConfiguration(
+            Configuration flinkConfig, CommandLine commandLine) {
+        Properties properties = commandLine.getOptionProperties(FLINK_CONFIG.getOpt());
+        LOG.info("Dynamic flink config items found: {}", properties);
+        for (String key : properties.stringPropertyNames()) {
+            String value = properties.getProperty(key);
+            if (StringUtils.isNullOrWhitespaceOnly(key)
+                    || StringUtils.isNullOrWhitespaceOnly(value)) {
+                throw new IllegalArgumentException("Illegal argument for key or value");
```

Comment:
Maybe we can print out the questionable key / value pair in command line to improve traceability.

Reply:
done. thanks for comment